cmd/k8s-operator/crdsforhelm/crdsforhelm.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !plan9 && !windows
+
+package crdsforhelm
```

Comment:
We'd probably still need a `main` to call this from corp, right?

Reply:
@knyar this was getting too complicated for the simple task. I have restructured the whole of this to both add the CRD to the chart and add the CRD to the static manifest by adding a bit more to the existing ./cmd/k8s-operator/generate which we can then also call from build infra.